storage/tests/system.py

```diff
@@ -82,6 +82,15 @@ def tearDownModule():
     retry(Config.TEST_BUCKET.delete)(force=True)
 
 
+class TestClient(unittest.TestCase):
+
+    def test_get_service_account_email(self):
+        expected = '{}@gs-project-accounts.iam.gserviceaccount.com'.format(
+            Config.CLIENT.project)
```

Comment:
New projects will use `service-[PROJECT-NUMBER]@gs-project-accounts.iam.gserviceaccount.com` May want to keep this loose for the future.